packages/components/src/FieldMessage/_docs/FieldMessage.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { FieldMessage } from "../index"
+
+const meta = {
+  title: "KAIO-staging/FieldMessage",
+  component: FieldMessage,
+  args: {
+    message:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullasemper odio vitae sem gravida rutrum.",
+  },
+} satisfies Meta<typeof FieldMessage>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {
+  parameters: {
+    docs: {
+      canvas: {
+        sourceState: "shown",
+      },
+    },
+  },
+}
+
+export const HelperText: Story = {
+  args: {
+    message:
+      "Your email address will be used to send survey notifications and reports",
+  },
+}
+export const DetailedHelperText: Story = {
+  args: {
+    message: (
+      <div>
+        Your email address will be used to send survey notifications and
+        reports. Learn more about <a href="/">how your data will be used</a>
+      </div>
+    ),
+  },
+}
+
+export const Error: Story = {
+  args: { status: "error", message: "Invalid email provided." },
+}
+
+export const Cautionary: Story = {
+  args: {
+    status: "caution",
+    message: "Do not share your password or account details.",
+  },
+}
+
+export const Success: Story = {
+  args: { status: "success", message: "Your account has been created" },
+}
+
+export const Reversed: Story = {
+  args: { reversed: true, status: "error" },
+  parameters: { backgrounds: { default: "Purple 700" } },
+  name: "reversed",
+}
+
+export const Position: Story = {
+  args: { position: "top" },
+}
```

Comment:
I'd probably render two `FieldMessage` side by side to show the default vs the top versionss

Reply:
Good idea, I've updated the story to have them side by side. I used a cautionary status since its visually easier to see than just whitespace